frontend/src/app/visualization/ngram/ngram.component.ts

```diff
@@ -64,6 +63,15 @@ export class NgramComponent extends ParamDirective implements OnChanges {
         paramService: ParamService
     ) {
         super(route, router, paramService);
+        this.currentParameters = new NgramParameters(
+            this.sizeOptions[0].value,
+            this.positionsOptions[0].value,
+            this.freqCompensationOptions[0].value,
+            'none',
+            this.maxDocumentsOptions[0].value,
+            this.numberOfNgramsOptions[0].value,
+            'date'
+        );
```

Comment:
I would expect the initial values to be based on the route. Does this work when loading the page with parameters in the route?

Reply:
Hm, good point!

Reply:
I checked once more to be sure, and indeed, it works as I'd intended: the constructor sets the defaults, and later in the component's lifetime, the route settings will be applied, if present.